assets/api-swagger/swagger.json

```diff
@@ -3183,6 +3183,10 @@
             "description": "Data passed in this object will be made available as dynamic content which you can merge into the template before sending.",
             "default": "{}"
           },
+          "url_parameters": {
```

Comment:
Maybe also update the example with the new URL parameters - https://github.com/Ometria/ometria.docs.assets.external/blob/fd537477a255aeb06df2c39a5ff05773e0d06323/assets/api-swagger/swagger.json#L3196

Reply:
Updated the example